core/src/test/scala/org/apache/spark/JobCancellationSuite.scala

```diff
@@ -59,7 +59,7 @@ class JobCancellationSuite extends SparkFunSuite with Matchers with BeforeAndAft
   }
 
   test("local mode, FIFO scheduler") {
-    val conf = new SparkConf().set(SCHEDULER_MODE, "FIFO")
+    val conf = new SparkConf().set(SCHEDULER_MODE.key, "FIFO")
```

Comment:
Ur, is this a breaking change, @yaooqinn ?

Reply:
It seems that I missed this in Scala version PR. I'm curious if we can avoid this kind of programing behavior change.